pex/variables.py

```diff
@@ -68,40 +74,47 @@ def _get_kv(cls, variable):
             return kv
 
     def __init__(self, environ=None, rc=None, use_defaults=True):
+        # type: (Optional[Dict[str, str]], Optional[str], bool) -> None
         self._use_defaults = use_defaults
-        self._environ = (environ if environ is not None else os.environ).copy()
+        self._environ = environ.copy() if environ is not None else os.environ.copy()
```

Comment:
MyPy was getting confused trying to find the common denominator between our `Dict` vs. the `_OSEnviron` type, then complaining that `.copy()` wasn't define on that common interface of `MutableMapping`. This fixes it.